### ISSUE TITLE
Allow filtering on missing variables

### ISSUE THREAD

**Issue by RealOrangeOne** [rcoh/angle-grinder#140]
Especially useful when parsing lines of different structures with `logfmt`, it's useful to say something like "only use lines where this variable exists", or something.

This could be something where missing variables are falsy, so using `where` would work as normal. Or perhaps a different operator, or a new filter entirely (eg `has`). For an intuitive feel, making them falsey, or never equal to anything in a `where` would be really nice, although may not be backwards compatible.

Currently I add a `contains` before `logfmt`, which feels like a bit of a hack.

### Example

```
type=event value=50
path=/ response_time=50ms
```

Using `| where value > 50` would result in an error for the second line. Whereas being able to do `| has value ` before it would filter out any of those.

**Comment by rcoh** [comment]
Yeah this is a good idea. I would make a unary function called has:

```
| where has(foo) | where foo > 50
```
